views/layout.erb

```diff
@@ -43,7 +43,7 @@
     <script src="https://cdn.jsdelivr.net/npm/tom-select@2.2.2/dist/js/tom-select.complete.min.js"></script>
 
     <!-- MathJax with mhchem extension -->
-    <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
+    <!-- <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> -->
```

Comment:
This line has been removed in the `main` branch, rebase it into this one before merging.